invenio_rdm_records/utils.py

```diff
@@ -23,7 +23,12 @@ def get_or_create_user(email):
             username=email.split("@")[0],
             password=hash_password("123456"),
             active=True,
-            preferences=dict(visibility="public", email_visibility="public"),
+            preferences=dict(
+                visibility="public",
+                email_visibility="public",
+                locale="en_GB",
```

Comment:
I suppose this only used to create users via cli or the admin user, but the locale and timezone are supposed to be hardcoded? 
Locale is set to `en` in other places, is there any reason why it's set to `en_GB` here?

Reply:
Fixed thx!